### PR TITLE
[DAE-77] Release 1.0.3

### PR DESCRIPTION
**Pull request by LucasMMota**
## Why? :open_book:
Releasing version 1.0.3

## What? :wrench:
- https://github.com/quintoandar/hive-metastore-client/pull/37 - Handling exception when adding duplicate partitions
- https://github.com/quintoandar/hive-metastore-client/pull/39 - Remove method returning for create_database_if_not_exists

## Type of change :file_cabinet:
- [X] Release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 
 Preferably use **Added**, **Changed**, **Removed** and **Fixed** topics in each release or unreleased log for a better organization.
 
+## [1.0.3](https://github.com/quintoandar/hive-metastore-client/releases/tag/1.0.3)
+### Changed
+* Handling exception when adding duplicate partitions ([#37](https://github.com/quintoandar/hive-metastore-client/pull/37))
+* Remove method returning for create_database_if_not_exists ([#39](https://github.com/quintoandar/hive-metastore-client/pull/39))
```
Comment (kenjihiraoka) [inline CHANGELOG.md:9]: Isn't better to use `Removed`?

Comment (LucasMMota) [reply CHANGELOG.md:9]: You mean this?
```suggestion
* Removed method returning for create_database_if_not_exists ([#39](https://github.com/quintoandar/hive-metastore-client/pull/39))
```
Or using the ### Remove?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 
 Preferably use **Added**, **Changed**, **Removed** and **Fixed** topics in each release or unreleased log for a better organization.
 
+## [1.0.3](https://github.com/quintoandar/hive-metastore-client/releases/tag/1.0.3)
+### Changed
+* Handling exception when adding duplicate partitions ([#37](https://github.com/quintoandar/hive-metastore-client/pull/37))
```
Comment (ribaldorafael) [inline CHANGELOG.md:8]:
```suggestion
* Handled exception when adding duplicate partitions ([#37](https://github.com/quintoandar/hive-metastore-client/pull/37))
```

File: CHANGELOG.md (hunk 1 above)
Comment (ribaldorafael) [inline CHANGELOG.md:9]: besides what @kenjihiraoka suggested:
 ```suggestion
* Removed method return for create_database_if_not_exists ([#39](https://github.com/quintoandar/hive-metastore-client/pull/39))
```

Comment (LucasMMota) [reply CHANGELOG.md:9]: Got it!

Comment (kenjihiraoka) [reply CHANGELOG.md:9]: Use the first one

**Review by kenjihiraoka — APPROVED** [review]
lgtm

**Review by felipemiquelim — APPROVED** [review]

**Pull request merged**